py-polars/polars/dataframe/frame.py

```diff
@@ -363,7 +363,7 @@ def __init__(
                 nan_to_null=nan_to_null,
             )
 
-        elif isinstance(data, (list, tuple, Sequence)):
+        elif isinstance(data, Sequence):
```

Comment:
This change removes redundancy in type checking during schema processing. `List | Tuple` are both type `Sequence`.
I found it when backtracking following the schema processing pipeline.

Reply:
If I recall correctly this may have been a micro-optimisation for some of our most latency-sensitive users, as the Sequence check is more expensive but the common-case is vanilla list/tuple 🤔

Reply:
Cool, yes you're right.